### PR TITLE
Hott 442 circle ci deploy

### PR DESCRIPTION
**Pull request by mafonso**
### Jira link

HOTT-422

### What?

I have added/removed/altered:

- [ ] Add blue/green deployment on CircleCI
- [ ] Decommission deprecated and unmaintained CF blue-green-deployment plugin used in GH actions
- [ ] Test blue deployment for availability before proceeding 

### Why?

I am doing this because:
- We are porting Github actions to CircleCI
- We are eliminating tech-debt in the process. 

### Have you? (optional)

- [ ] Reviewed view changes with stake holders
- [ ] Validated mobile responsive behaviour of view changes

### Deployment risks (optional)

- Makes changes to our complex routing setup that may affect apis to proxying to backend

### CODE REVIEW

File: .circleci/config.yml [hunk 1]
```diff
@@ -44,19 +128,63 @@ jobs:
...
+          type: approval
+          requires:
+            - deploy_dev
+      - deploy_staging:
+          context: trade-tariff
+          filters:
+            branches:
+              only:
+                - master
+                - HOTT-442_CircleCI-Deploy
```
Comment (chriscoffee) [inline .circleci/config.yml]: Needs to be removed? 👀

Comment (mafonso) [reply .circleci/config.yml]: Good spot. Missed that one.

Comment (chriscoffee) [reply .circleci/config.yml]: 🍾

File: .circleci/config.yml [hunk 2]
```diff
@@ -5,6 +5,86 @@ orbs:
...
+          name: "Setup CF CLI"
+          command: |
+            curl -L -o cf.deb 'https://packages.cloudfoundry.org/stable?release=debian64&version=7.2.0&source=github-rel'
+            sudo dpkg -i cf.deb
+            cf -v
+            cf api "$CF_ENDPOINT"
+            cf auth "$CF_USER" "$CF_PASSWORD"
+            cf add-plugin-repo CF-Community https://plugins.cloudfoundry.org
+            cf install-plugin app-autoscaler-plugin -r CF-Community -f
+            cf target -o "$CF_ORG" -s "<< parameters.space >>"
```
Comment (tavyy) [inline .circleci/config.yml:29]: Interesting.

**Review by tavyy — APPROVED** [review]
I'm happy with it.

**Pull request merged**